audition/distance_from_best.py

```diff
@@ -225,7 +239,7 @@ def generate_plot_data(
                 )
                 SELECT dist.model_group_id, pct_diff, mg.model_type,
                        COUNT(*) AS num_models,
-                       AVG(CASE WHEN below_best <= pct_diff THEN 1 ELSE 0 END) AS pct_of_time
+                       AVG(CASE WHEN dist_from_best_case <= pct_diff THEN 1 ELSE 0 END) AS pct_of_time
```

Comment:
It depends if:

1. We care more about the theoretical max/min, in which case I think we should do it in catwalk.metrics/allow it to be decorated

2. or the observed max/min, in which case I think we should infer and save the exact values, I guess in the object? I assume the padding would be something we handle when graphing, and we can automatically figure that out.